python/ovs_module/containerovsdb/ovsdb_lib.py

```diff
@@ -16,9 +16,12 @@ def add_ovs_bridge(self, bridge, datapath_type):
         return self.ovsdb.add_br(bridge, may_exist=True,
                                  datapath_type=datapath_type).execute()
 
+    def _set_mtu_request(self, dev, mtu):
```

Comment:
why you do't use https://github.com/openstack/os-vif/blob/4e6c16037c1139e189d2ec7b6a90b43c3b10b16b/vif_plug_ovs/ovsdb/ovsdb_lib.py#L50

Reply:
So I'm using it but only for the related case (vdpa port)  https://github.com/openstack/os-vif/blob/4e6c16037c1139e189d2ec7b6a90b43c3b10b16b/vif_plug_ovs/ovsdb/ovsdb_lib.py#L63
Or do you need me to do it in general ? @moshe010